main.py

```
@@ -181,5 +181,64 @@ def clear_cache():
+        click.echo("✅ Digest built successfully!")
+        click.echo(f"📄 JSON saved to: {json_path}")
+        
+        # Defensive access to metadata
+        metadata = digest.get('metadata', {})
+        total_clips = metadata.get('total_clips', 0)
+        total_events = metadata.get('total_events', 0)
+        click.echo(f"📊 Summary: {total_clips} clips, {total_events} events")
+        
+    except FileNotFoundError as e:
+        click.echo(f"❌ {e}")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)